### ISSUE TITLE
📝Create basic MovieCard component

### ISSUE THREAD

**Issue by Thenlie** [Thenlie/Streamability#59]
# Movie Card Component

The movie card will take date for an individual show from The MovieDB and output a card of that show. A mockup of the card can be seen below. The show data should be passed into the component so it can be reused across the entire app.

## 📸 Mockup

![Image](https://user-images.githubusercontent.com/41388783/200218778-23718ac5-1ee6-4aaf-bdee-62a055bb3855.png)

**Comment by Thenlie** [comment]
The flip animation/effect could be created as a new ticket. Additionally, if we end up creating a placeholder component with functional data but incomplete styles, be sure to create a new ticket to update the styles once the mockup is finished.

**Comment by Thenlie** [comment]
Closed by #85
